src/xdp-utils.c

```diff
@@ -1028,7 +1028,7 @@ xdp_filter_options (GVariant *options,
             }
         }
 
-      g_variant_builder_add (filtered, "{sv}", supported_options[i].key, g_steal_pointer (&value));
+      g_variant_builder_add (filtered, "{sv}", supported_options[i].key, value);
```

Comment:
Two questions:

 * Isn't `value` a floating variant?
 * Doesn't `g_variant_builder_add` consume floating variants?

GVariant programming always tricks and annoys me :slightly_frowning_face:

Reply:
1. `g_variant_lookup_value` is "transfer-full".
2. since it's not floating it is ref().